src/main/java/host/caddy/services/YelpSearch.java

```diff
@@ -0,0 +1,61 @@
+package host.caddy.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import host.caddy.models.yelp.Yelp;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class YelpSearch {
+    private Yelp yelp = new Yelp();
+    private RestTemplate restTemplate = new RestTemplate();
+    private MultiValueMap<String, String> headers = new LinkedMultiValueMap<>();
+    private Map map = new HashMap<String, String>();
+    private Gson gson = new Gson();
+
+
+
+    public Yelp byName(String name, String lat, String lon) {
+        query(name,lat,lon);
+        return yelp;
+    }
+
+    public String byNameJSON(String name, String lat, String lon) {
+        query(name,lat,lon);
+        return gson.toJson(yelp, Yelp.class);
+    }
+
+
+    private String query(String name, String lat, String lon){
+            map.put("Content-Type", "application/graphql");
+            map.put("Authorization", "Bearer SOmPECWYjuUJavKVazVl9GRrX9XJF0uaSe3n7361trmT2e_5BCgJucbNTg-o-VurkN2c60qZqOSylvtRrmw1njqDnEVZvzyZNShCMHXYE9U542-Fe4O3uJPFFkRtWnYx");
```

Comment:
This bearer token shouldn't be here. There's an example in this [lesson](http://java.codeup.com/spring/extra-features/file-upload/) on how to use the `application.properties` file to avoid hard-coding it.

Everithing else looks great!